src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -1,5 +1,6 @@
 /*
- * This file is part of the TrinityCore Project. See AUTHORS file for Copyright information
+ * Copyright (C) 2008-2019 TrinityCore <https://www.trinitycore.org/>
+ * Copyright (C) 2006-2009 ScriptDev2 <https://scriptdev2.svn.sourceforge.net/>
```

Comment:
could you explain a bit more in detail the reason for these 2 line changes ?

Reply:
for sure he is using outdated base.

Reply:
I have no idea where they come from. Let me check on those. I've had this script around for a while though.

Reply:
Wow .. never even saw them when i looked at the changes. I must be blind. Those two can be removed.